libs/@blockprotocol/graph/src/types/entity.ts

```diff
@@ -52,7 +52,7 @@ export type EntityPropertiesObject = {
   [_: BaseUri]: EntityPropertyValue;
 };
 
-type HalfClosedInterval = TimeInterval<
+type HalfClosedInterval = TimeIntervalUnresolved<
```

Comment:
Shouldn't this be simply `TimeInterval`? It does not have `null` in it.

Reply:
That makes more sense (good spot), both equivalent of course, but that's the problem with structural types 😭 

[Replace unnecessary usages of TimeIntervalUnresolved](https://github.com/blockprotocol/blockprotocol/pull/928/commits/d4abfd57510ebdf2ce7eaf7b6f276afb5c98f030)